### DISCUSSION TITLE
Any way to forbid wifi hotspot?

### DISCUSSION THREAD

**Discussion by nekopep** [FoobarOy/foomuuri#43] [Q&A] [answered]
Today I discovered that one of my user was using it's wifi wlan device as a passwordless hotspot... allowing anybody to connect to enterprise network :/
I checked how to prevent wifi hotspot creation, but did not find any clue on that topic so far, and it is a functionnality that is literally one click away from user...
Do you think a foomuuri config could block packet from created hotspot as a workaround?

**Comment by kimheino** [comment]
Firewall (Foomuuri on any other) can't protect against rogue hotspot running somewhere in your internal lan. 

Foomuuri can protect hotspot running on same computer (or simply turn ip_forwarding off) but I guess your hotspot-computer wasn't running Foomuuri.

If your internal lan is static and has only few computers you could do mac address filtering. It's not viable solution for dynamic or large lans.

**Reply by nekopep** [reply]
I want to protect hotspot running on same computer to be clear.
Do you know how to do it?
I've added :
net.ipv4.ip_forward = 0
net.ipv4.conf.all.forwarding = 0
net.ipv6.conf.all.forwarding = 0
to sysctl.conf, but nm-applet seems to reactivate it :/

**Reply by nekopep** [reply]
So that why I was looking at foomuuri option as I plan to install on all computers.

**Reply by kimheino** [reply, marked as answer]
> I want to protect hotspot running on same computer to be clear.

Ok, then Foomuuri can do it.

> sysctl.conf, but nm-applet seems to reactivate it :/

Then it's better to simply filter traffic. Try adding this to your config:

```
forward {
  drop log
}
```

That drops all forwarded packets. You can be more specific if needed:

```
forward {
  iifname eth0 oifname wlan0 drop log  # eth0 -> wlan0
  iifname wlan0 oifname eth0 drop log  # wlan0 -> eth0
}
```

**Reply by nekopep** [reply]
Works perfectly :+1: 
Foomuuri is definitively a very good and simple firewall rule manager :smile:
